backend/app/api/v1/admin.py

```
@@ -66,17 +67,27 @@ async def upload_report_template(
     admin: User = Depends(require_admin),
-    db: AsyncSession = Depends(get_db),
-):
+    db: AsyncSession = Depends(get_db)):
     try:
-        return await admin_service.replace_clinical_protocols(
-            db=db,
-            files=files,
-            uploaded_by_user_id=admin.id,
-        )
+        return await admin_service.add_clinical_protocols(db=db, file=file, uploaded_by_user_id=admin.id, docs_path="clinical_protocols")
     except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)